katsdpimager/templates/report.html.j2

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Spectral imager report</title>
+    {{ resources | safe | indent(4) }}
+    {{ script | safe | indent(4) }}
+    <style>
+      {{ static["bootstrap-4.4.1.min.css"] | indent(6) }}
+    </style>
+  </head>
+  <body>
+    <div class="container-fluid">
+{% for target in targets %}
+      <div class="row">
+        <div class="col">
+          <h1>{{ target.name }}</h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-auto">
+          <div class="card">
+            <h5 class="card-header">Properties</h5>
+            <table class="table">
+              <tbody>
+                <tr>
+                  <td>Description</td>
+                  <td>{{ target.description }}</td>
+                </tr>
+                <tr>
+                  <td>Antennas</td>
+                  <td>{{ common.antennas | count }}</td>
+                </tr>
+                <tr>
+                  <td>Time on target</td>
+                  <td>{{ target.time_on_target | duration }}</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+        <div class="col-auto">
+          <div class="card">
+            <h5 class="card-header">Channels</h5>
+            <table class="table">
+              <!-- <colgroup><col style="width: 10em"><col></colgroup> -->
+              <tbody>
+                <tr>
+                  <td>Successful</td>
+                  <td>{{ target.status|select("equalto", "complete")|list|count }}</td>
+                </tr>
+                <tr>
+                  <td>Flagged</td>
+                  <td>{{ target.status|select("equalto", "no-data")|list|count }}</td>
+                </tr>
+                <tr>
+                  <td>Masked</td>
+                  <td>{{ target.status|select("equalto", "masked")|list|count }}</td>
+                </tr>
+                <tr>
+                  <td>Failed</td>
+                  <td>{{ target.status|select("equalto", "failed")|list|count }}</td>
+                </tr>
+                <tr>
+                  <td>Total</td>
+                  <td>{{ common.channels }}</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <div class="card">
+            <h5 class="card-header">Processing status</h5>
+            {{ target.plots['status'] | safe | indent(12) }}
+          </div>
+          <div class="card">
+            <h5 class="card-header">Flux density</h5>
```

Comment:
Perhaps call this 'Signal to noise' rather than 'Flux density'

Reply:
Done.